plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/TestHiveParquetWithBloomFilters.java

```diff
@@ -0,0 +1,179 @@
+            assertQueryStats(
+                    getSession(),
+                    String.format(String.format("SELECT * FROM %s WHERE %s IN (%s)", tableName, COLUMN_NAME, NOT_EXIST_VALUE)),
+                    queryStats -> {
+                        assertThat(queryStats.getPhysicalInputPositions()).isEqualTo(0);
+                        assertThat(queryStats.getProcessedInputPositions()).isEqualTo(0);
+                    },
+                    results -> assertThat(results.getRowCount()).isEqualTo(0));
+
+            // when enabling reading bloomfilter, row groups are not pruned when searching for an exist value
+            assertQueryStats(
+                    getSession(),
+                    String.format(String.format("SELECT * FROM %s WHERE %s IN (%s)", tableName, COLUMN_NAME, TEST_VALUES.get(0))),
+                    queryStats -> {
+                        assertThat(queryStats.getPhysicalInputPositions()).isGreaterThan(0);
+                        assertThat(queryStats.getProcessedInputPositions()).isEqualTo(queryStats.getPhysicalInputPositions());
+                    },
+                    results -> assertThat(results.getRowCount()).isEqualTo(1));
+
+            // when disabling reading bloomfilter, row groups are not pruned when searching for a non-exist value
+            assertQueryStats(
+                    disableBloomFilters(getSession()),
+                    String.format(String.format("SELECT * FROM %s WHERE %s IN (%s)", tableName, COLUMN_NAME, NOT_EXIST_VALUE)),
+                    queryStats -> {
+                        assertThat(queryStats.getPhysicalInputPositions()).isGreaterThan(0);
+                        assertThat(queryStats.getProcessedInputPositions()).isEqualTo(queryStats.getPhysicalInputPositions());
+                    },
+                    results -> assertThat(results.getRowCount()).isEqualTo(0));
+        }
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        return HiveQueryRunner.builder()
+                .addHiveProperty("parquet.use-bloom-filter", "true")
+                .build();
+    }
+
+    private static Session disableBloomFilters(Session session)
+    {
+        return Session.builder(session)
+                .setCatalogSessionProperty(session.getCatalog().orElseThrow(), "parquet_use_bloom_filter", "false")
+                .build();
+    }
+
+    private static void createParquetBloomFilterSource(File tempFile, String columnName, List<Integer> testValues)
+            throws Exception
+    {
+        List<ObjectInspector> objectInspectors = singletonList(javaIntObjectInspector);
+        List<String> columnNames = ImmutableList.of(columnName);
+
+        JobConf jobConf = new JobConf(newEmptyConfiguration());
+        jobConf.setEnum(WRITER_VERSION, PARQUET_1_0);
+        jobConf.setBoolean(BLOOM_FILTER_ENABLED, true);
```

Comment:
nit: Let's enable the bloom filter only for `columnName` and not for all columns